src/ucp/wireup/select.c

```diff
@@ -1011,7 +1011,8 @@ static double ucp_wireup_aux_score_func(const ucp_worker_iface_t *wiface,
 }
 
 static void ucp_wireup_fill_aux_criteria(ucp_wireup_criteria_t *criteria,
-                                         unsigned ep_init_flags)
+                                         unsigned ep_init_flags,
+                                         unsigned mandatory_flags)
```

Comment:
maybe use `uint64_t` to be consistent with `criteria->remote_iface_flags.mandatory` type

Reply:
Done. Force pushed changes.